lib/queuePopulator/QueuePopulator.js

```diff
@@ -236,14 +236,19 @@ class QueuePopulator {
     }
 
     _setupMetricsClients(cb) {
-        // Metrics Consumer
-        this._mConsumer = new MetricsConsumer(this.rConfig,
-            this.mConfig, this.kafkaConfig, metricsExtension);
-        this._mConsumer.start();
-
-        // Metrics Producer
-        this._mProducer = new MetricsProducer(this.kafkaConfig, this.mConfig);
-        this._mProducer.setupProducer(cb);
+        // TODO: Handle metrics for bucket notification
```

Comment:
Any TODO should have a ticket associated so we can track if we need more work
-> is it something we need to to in this ticket?

Reply:
As far as i understand this code is related to Redis metrics which are not used anymore (we are switching to prometheus) and will be removed in a later ticket. No need to forward port this TODO.